pkgs/applications/misc/gnome-secrets/default.nix

```diff
@@ -45,21 +45,19 @@ python3Packages.buildPythonApplication rec {
     glib
     gdk-pixbuf
     libadwaita
+    # Use same libpwquality from pythonPackages instead of redundant pkgs.libpwquality
+    # TODO: better way to get this output, or make meson not check for dependency?
+    (libpwquality.override { enablePython = true; inherit (python3Packages) python; })
```

Comment:
I think we should move this into top-level or where the package attr for gnome-secrets is defined. Having it inlined is problematic when it is used in multiple places.

Reply:
I did a little more investigation, it looks like customization.nix adds links to all a derivations outputs to each of it's output attrs, so I just use that.